### DISCUSSION TITLE
Android app?

### DISCUSSION THREAD

**Discussion by QwertyIsCoding** [official-stockfish/Stockfish#4252] [Q&A]
Is there going to be an Android app that users can install just like the IOS build? I don't find one on the website or Play Store... Or just a .APK build would do.

**Comment by XInTheDark** [comment]
I'm sure there are android ches GUIs that come with Stockfish.

For a start, the lichess mobile app has stockfish analysis installed in it.

There are also small apps like [this](https://play.google.com/store/apps/details?id=com.chessimprovement.chessis&hl=en&gl=US) and [this](https://play.google.com/store/apps/details?id=ccc.chess.engine.stockfish) which have Stockfish installed in it (not sure if they have the latest version or not).

**Comment by Joachim26** [comment]
There are, made by Archimedes, a lot of chess engine-apks including Stockfish and StockfishNPS:
https://sourceforge.net/projects/chess-engines-for-android-oex/files/
